migration/v2.18.0-ck8s1-v2.19.x-ck8s1/upgrade-cluster.md

```diff
@@ -2,6 +2,8 @@
 
 1. Checkout the new release: `git checkout v2.19.x-ck8s1`
 
+> The new kubespray version does not work with older teraform setups. If teraform actions are needed revert to v2.18.1.
```

Comment:
Maybe mention that if [this issue](https://github.com/elastisys/compliantkubernetes-kubespray/issues/176) is closed you can use that fix instead.